yaksh/documentation/moderator_docs/creating_course.rst

```diff
@@ -3,26 +3,29 @@ Courses
 =======
 
 For students to take a quiz, it is imperative for the moderator to create a course first. 
+A course contains modules and module contains lessons and quizzes.
```

Comment:
A course can contain several modules and a module can contains several lessons and/or quizzes.